src/ElectricPokemon.java

```diff
@@ -0,0 +1,97 @@
+import org.w3c.dom.ls.LSOutput;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private static final String type = "electric";
+    private final List<String> attacks = Arrays.asList("thunderpunch", "electroball", "thunder", "volttackle");
+
+
+//----- constructor ----------------------------------------------------------------------------------------------
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(type, name, level, hp, food, sound);
+    }
+
+
+//----- methodes ----------------------------------------------------------------------------------------------
+// aangezien in pokemon wiki niet wordt aangegeven hoeveel hp een pokemon verliest maar dit daar veel ingewikkelder ligt, heb ik mijn eigen systeem bedacht. Dus nu geef ik in de methode het aantal minhp mee (1, 2 of 3 heb ik bedacht) en afh van welk type tegen welk type geef ik een factor mee. Dus bijvoorbeeld: minHp  Het type fire🔥 doet de meeste schade aan grass-pokémons🌿 > factor 4, dus min 8 hp, daarna aan water-pokémons🌊 > factor 3 = min 6 hp, dan de electric-pokémons⚡ > factor 2 = min 4 hp en het minste bij fire-pokémons🔥 > factor 1 = min 2 hp.
+
+// onderstaande methode is een methode die ik in elk van de electricPokemons toepas. Deze is weer anders voor de WaterPokemon enz omdat de factor anders is.
+    public void templateAttackMethod(Pokemon name, Pokemon enemy, String nameAttack, int hpMinInAttack) {
```

Comment:
Heel goed dat je hier een methode hebt gedefinieerd in plaats van jezelf telkens te moeten herhalen. Je zou zelfs de factor in de parameters mee kunnen geven (mischien als een aparte kalsse of  enumeratie) zodat je deze methode ook voor andere typen pokemon kunt gebruiken.